migrate/cockroach.go

```diff
@@ -0,0 +1,187 @@
+package migrate
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/mattes/migrate"
```

Comment:
This is the first instance of a 3rd party dependency in this package. As such we should at very least vendor the dependencies and commit the vendor folder and preferably commit a dependency manifest via `dep`